server.js

```diff
@@ -40,7 +40,7 @@ wss.on('connection', function connection(ws) {
       //var message = JSON.parse(data);
       var message = gbTree.GBClientMessage.decode(data);
       console.log('received (parsed):');
-      console.dir(message, { depth: null });
+      console.dir(message, { depth: 3 });
```

Comment:
mind just adding a comment explaining why this is depth 3, along with our (inconclusive) findings from experimenting today? 

(basically just that the map object is complex and deep and seems to blow things up)